src/java/org/apache/nutch/crawl/CrawlDb.java

```diff
@@ -28,8 +28,12 @@
 import org.apache.hadoop.io.*;
```

Comment:
@Omkar20895 can you please sort out all instances of wildcard imports for Hadoop?